osmaxx-py/excerptconverter/converter_helper.py

```diff
@@ -7,6 +6,42 @@
 from osmaxx.excerptexport import models
 
 
+def module_converter_configuration(name, export_formats, export_options):
+    """
+    :param export_formats example:
+        {
+            'txt': {
+                'name': 'Text',
+                'file_extension': 'txt',
+                'mime_type': 'text/plain'
+            },
+            'markdown': {
+                'name': 'Markdown',
+                'file_extension': 'md',
+                'mime_type': 'text/markdown'
+            }
+        }
+    :param export_options example:
+        {
+            'image_resolution': {
+                'label': 'Resolution',
+                'type': 'number',
+                'default': '500'
+            },
+            'quality': {
+                'label': 'Quality',
+                'type': 'number',
+                'default': '10'
+            }
+        }
+    """
+    return {
```

Comment:
>>The function's signature enforces presence of expected components
> 
> No, they don't, they still can be passed as `None` and everything looks fine.

'enforces' might have been the wrong term here. What I meant, was: Using that function can guide us by its signature. If we later add another required component, we'll just expand the signature accordingly and all calls that need to up updated will be highlighted as errors in the IDE. Of course, a determined coder could still just not use the function or pass `None` or even nonsense to it.